angular/projects/spark-angular/src/lib/components/sprk-masthead/directives/sprk-masthead-branding/sprk-masthead-branding.directive.ts

```diff
@@ -0,0 +1,32 @@
+import { Directive, Input, HostBinding, ElementRef } from '@angular/core';
+
+@Directive({
+  selector: '[sprkMastheadBranding]',
+})
+export class SprkMastheadBrandingDirective {
+  /**
+   * @ignore
+   */
+  constructor(public ref: ElementRef) {}
+
+  /**
+   * The value supplied will be assigned to the `data-id` attribute on the
```

Comment:
**This applies to all Input descriptions in this PR.**

"The value supplied will be assigned to the..." is saying the same thing twice.

Consider using`The value assigned to the x attribute...` to make this even more readable.

Reply:
Great topic to discuss! We talked about this in chat and the recommendation is for @RVMendoza  to sync with Steph on creating an issue to review our standards for documenting props.